doc/how-to-build.md

```diff
@@ -106,12 +106,19 @@ Makefile rules:
 
 1. Install perl's package manager `cpanminus` first
 2. Then install `test-gninx` via `cpanm`:：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
-3. Clone source code：`git clone https://github.com/openresty/test-nginx.git`;
+3. Clone source code：`git clone https://github.com/apache/incubator-apisix.git`;
```

Comment:
The old version is right. The `apisix` and `test-nginx` should be put in the same folder.

```
$ ls 
apisix       test-nginx
```

Reply:
Hi @membphis  sure will change accordingly.